vips/vip-354/bscmainnet.ts

```diff
@@ -0,0 +1,62 @@
+import { ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+export const VUSDT_CORE = "0xfD5840Cd36d94D7229439859C0112a4185BC0255";
+export const VUSDC_CORE = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const VDAI_CORE = "0x334b3eCB4DCa3593BCCC3c7EBD1A1C1d1780FBF1";
+export const VFDUSD_CORE = "0xC4eF4229FEc74Ccfe17B2bdeF7715fAC740BA0ba";
+export const VUSDT_STABLECOINS = "0x5e3072305F9caE1c7A82F6Fe9E38811c74922c3B";
+export const VUSDT_GAMEFI = "0x4978591f17670A846137d9d613e333C38dc68A37";
+export const VUSDT_DEFI = "0x1D8bBDE12B6b34140604E18e9f9c6e14deC16854";
+
+export const RATE_MODELS = {
+  JumpRateModel_base0bps_slope875bps_jump25000bps_kink8000bps: "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D",
+  JumpRateModel_base0bps_slope875bps_jump50000bps_kink8000bps: "0xE19C14171C2aC6CA63E008133e4B0D8571164BA3",
+  JumpRateModelV2_base0bps_slope1000bps_jump25000bps_kink8000bps: "0x2ba0F45f7368d2A56d0c9e5a29af363987BE1d02",
+  JumpRateModelV2_base200bps_slope1500bps_jump25000bps_kink8000bps: "0x5ECa0FBBc5e7bf49dbFb1953a92784F8e4248eF6",
+
+  JumpRateModel_base0bps_slope1000bps_jump25000bps_kink8000bps: "0x62A8919C4C413fd4F9aef7348540Bc4B1b5CC805",
+  JumpRateModel_base0bps_slope1000bps_jump50000bps_kink8000bps: "0x05c68eE7c9c66BE3a42476fe3838DE65ddE968f1",
+  JumpRateModelV2_base0bps_slope375bps_jump25000bps_kink8000bps: "0xb36b273601Ac5e0CaBD0845b7B8caa3426611Ca0",
+  JumpRateModelV2_base200bps_slope1350bps_jump25000bps_kink8000bps: "0x3aB2e4594D9C81455b330B423Dec61E49EB11667",
+};
+
+const vip354 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-354 Risk Parameters Adjustments",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      ...[VUSDT_CORE, VUSDC_CORE, VDAI_CORE].map((vToken: string) => ({
+        target: vToken,
+        signature: "_setInterestRateModel(address)",
+        params: [RATE_MODELS.JumpRateModel_base0bps_slope1000bps_jump25000bps_kink8000bps],
+      })),
+      {
+        target: VFDUSD_CORE,
+        signature: "_setInterestRateModel(address)",
+        params: [RATE_MODELS.JumpRateModel_base0bps_slope1000bps_jump50000bps_kink8000bps],
```

Comment:
The multiplier of this IR contract is 0.1, but we need a IR with multiplier 0.075, according to the [Chaos labs recommendations](https://community.venus.io/t/chaos-labs-interest-rate-curve-amendment-for-stablecoins-on-venus/4531)

<img width="665" alt="Screenshot 2024-08-21 at 13 17 49" src="https://github.com/user-attachments/assets/c9109ab0-ec2b-46a9-9fbd-9e22ffcb1e1e">